finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/Sentinels.scala

```diff
@@ -0,0 +1,181 @@
+package com.twitter.finagle.redis.protocol.commands
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.protocol.Commands._
+import com.twitter.finagle.redis.util._
+import com.twitter.finagle.redis.ClientError
+
+import java.net.InetSocketAddress
+
+import org.jboss.netty.buffer.ChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffers
+
+case class SentinelMaster(name: ChannelBuffer)
+  extends Sentinel(SentinelMaster.channelBuffer, Seq(name))
+
+object SentinelMaster extends SentinelHelper {
+  val command = "MASTER"
+  def apply(args: Seq[Array[Byte]]): SentinelMaster = {
+    val name = trimList(args, 2, "SENTINEL MASTER")(0)
+    new SentinelMaster(ChannelBuffers.wrappedBuffer(name))
+  }
+}
+
+case class SentinelMasters() extends Sentinel(SentinelMasters.channelBuffer, Nil)
```

Comment:
They both have a "command" property, with different values. Instead of using a different name, can we just use a dummy param?

    case class SentinelMasters(dummy: Int = 0) extends Sentinel(SentinelMasters.channelBuffer, Nil)

    object SentinelMasters extends SentinelHelper {
      val command = "MASTERS"
      def apply(args: Seq[Array[Byte]]): SentinelMasters = {
        new SentinelMasters()
      }
    }